lib/hem/tasks/selenium.rb

```diff
@@ -2,15 +2,17 @@
 namespace :selenium do
   require_relative 'selenium/chromedriver'
   require_relative 'selenium/install'
+  require 'hem/lib/selenium/jar'
 
   desc 'Start Selenium'
   task :start do
+    jarPath = Hem::Selenium::Jar::getLocalPath();
     cmd  = [
       'export NSPR_LOG_MODULES=all:3;',
-      "java -jar './bin/selenium-server-standalone.jar'",
+      'java ',
         '-Dwebdriver.firefox.logfile=/tmp/selenium-firefox.log',
         '-Dwebdriver.chrome.driver=./bin/chromedriver',
-        '-trustAllSSLCertificates',
```

Comment:
@kierenevans I did research in this topic and I found that there is a browser level setting (capability) which can turn off the verification, see https://github.com/Behat/MinkExtension/blob/master/src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php#L113
So in behat.yml should look like
```yaml
extensions:
    Behat\MinkExtension\ServiceContainer\MinkExtension:
      sessions:
        firefox:
          selenium2:
            wd_host: http://localhost:4444/wd/hub
            browser: firefox
            acceptSslCerts: true
```
Therefore I would not bother with turning it off on a global level.